integration-tests/hibernate-orm-mongodb-panache/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.url=jdbc:h2:tcp://localhost/mem:test
+quarkus.datasource.jdbc.max-size=8
+
+quarkus.hibernate-orm.dialect=org.hibernate.dialect.H2Dialect
+quarkus.hibernate-orm.database.generation=drop-and-create
+
+quarkus.hibernate-orm.statistics=true
+quarkus.hibernate-orm.metrics.enabled=true
+
+quarkus.mongodb.connection-string=mongodb://localhost:27018,localhost:27019
+quarkus.mongodb.write-concern.journal=false
+quarkus.mongodb.database=books
+
+# fake a different MongoDB instance
+quarkus.mongodb.cl2.connection-string=mongodb://localhost:27018,localhost:27019
+quarkus.mongodb.cl2.write-concern.journal=false
```

Comment:
Keep it simple, don't test different MongoDB instance

Reply:
Fixed in f470047